hosts/tunnel.go

```diff
@@ -34,7 +36,11 @@ func (h *Host) TunnelUp(ctx context.Context, dialerFactory DialerFactory, cluste
 	}
 	// set Docker client
 	logrus.Debugf("Connecting to Docker API for host [%s]", h.Address)
-	h.DClient, err = client.NewClient("unix:///var/run/docker.sock", DockerAPIVersion, httpClient, nil)
+	h.DClient, err = client.NewClientWithOpts(
```

Comment:
What happened here ?

Reply:
NewCLient is going to be deprecated and basically now it calls NewClientWithOpts, NewClientWithOpts supports configuring the dialer timeout which was causing a very big delay ~ 2 min every time rke tries to connect to a failed node